code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/interfaces/LDAPConfig.java

```diff
@@ -44,4 +48,6 @@
 
     long getConnectionTimeout();
 
+    List<Identity> getAllowedIdentities();
```

Comment:
thats what I suspected, thanks! Would be nice to add the doc annotation at some point